google/cloud/bigquery/client.py

```diff
@@ -2197,15 +2197,16 @@ def load_table_from_dataframe(
         else:
             job_config = job.LoadJobConfig()
 
-        if job_config.source_format:
-            if job_config.source_format != job.SourceFormat.PARQUET:
-                raise ValueError(
-                    "Got unexpected source_format: '{}'. Currently, only PARQUET is supported".format(
-                        job_config.source_format
-                    )
-                )
-        else:
+        supported_formats = (job.SourceFormat.CSV, job.SourceFormat.PARQUET)
```

Comment:
Nit: we can use a set literal here.

```suggestion
        supported_formats = {job.SourceFormat.CSV, job.SourceFormat.PARQUET}
```